### PR TITLE
enable container log rotation

### LINKED ISSUE #1214: Docker containers log rotation
Log rotation should be set on nodes managed by InfraKit.
```
find /var/lib/docker/containers/ -type f -name \*.log -size +1G -exec ls -lh {} \;
-rw-r----- 1 root root 4.5G May 18 23:06 /var/lib/docker/containers/a9bda05095de19157d8b7ded2bcd6a6f68e0d4fdf71dd55e6d4eb40aabc0621a/a9bda05095de19157d8b7ded2bcd6a6f68e0d4fdf71dd55e6d4eb40aabc0621a-json.log
-rw-r----- 1 root root 1.5G May 15 07:20 /var/lib/docker/containers/802d5cfe0606ae5875514d9b8ff4785b243ecf923257a6590e14acfb14664557/802d5cfe0606ae5875514d9b8ff4785b243ecf923257a6590e14acfb14664557-json.log
```
This examples are related to nats (4.5G) and etcd (1.5G).

This can be easily fixed by configuring _log-opts_ in the `daemon.json` file.

### PR DESCRIPTION
**Pull request by ndegory**
Fix #1214 

Add container log rotation on cluster nodes

### CODE REVIEW

File: platform/bootstrap/userdata-aws [hunk 1]
```diff
@@ -10,6 +10,7 @@ DRAIN_MANAGER=${DRAIN_MANAGER:-false}
 OVERLAY_NETWORKS=${OVERLAY_NETWORKS:-ampnet}
 #MIRROR_REGISTRIES=
 #DOCKER_DEVICE=/dev/sdn
+SYSTEMD_DOCKER_OVERRIDE=/etc/systemd/system/docker.service.d/docker.conf
 
 _install_docker(){
   wget -qO- https://get.docker.com/ | sh || return 1
```
Comment (subfuzion) [inline platform/bootstrap/userdata-aws:16]: Just a note for now, but we're probably going to want to ensure we install specific Docker versions pretty soon.

**Comment by bquenin** [comment]
Using `systemd` logging driver will break both our logs collection and the possibility to use `docker logs` command as stated here: https://docs.docker.com/engine/api/v1.28/#operation/ContainerLogs.

**Comment by subfuzion** [comment]
@bquenin systemd logging driver?

**Review by subfuzion — APPROVED** [review]
LGTM (except for the `systemd` log driver ... kidding!) ;)  (inside joke for anyone else reading)

**Pull request merged**